tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -304,6 +304,38 @@ describe(__filename, () => {
       .toHaveProp('disabled', true);
   });
 
+  it('trims leading and trailing spaces from slug and name before submitting', () => {
+    const authorUsername = 'collection-owner';
+    const name = 'trishul';
+    const slug = 'trishul';
+    const lang = 'en-US';
+    dispatchSignInActions({ lang, store });
+
+    const collection = createInternalCollection({
+      detail: createFakeCollectionDetail({ authorUsername, name, slug }),
+    });
+
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const root = render({ collection });
+
+    // Enter in collection name and slug with trailing and leading spaces.
+    typeInput({ root, name: 'name', text: '  trishul   ' });
+    typeInput({ root, name: 'slug', text: '  trishul   ' });
```

Comment:
The same goes here, by using a template string with `${slug}` in place of `'  trishul   '`.